core/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/status/SnapshotStatus.java

```diff
@@ -162,9 +161,7 @@ public static SnapshotStatus readSnapshotStatus(StreamInput in) throws IOExcepti
     public String toString() {
         try {
             XContentBuilder builder = XContentFactory.jsonBuilder().prettyPrint();
-            builder.startObject();
```

Comment:
There's utility method that could replace all of this in `org.elasticsearch.common.String`. I think `toString(ToXContent toXContent, boolean pretty, boolean human)` can replace all of this methods code.

Reply:
Thanks for reviewing this.